src/components/SfacExtensionSellNotification.js

```diff
@@ -33,15 +39,28 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
   },
 }))
-const SfacExtensionSellNotification = ({ onNo, onYes }) => {
+const SfacExtensionSellNotification = ({ browser, userId }) => {
   const [open, setOpen] = useState(true)
   const classes = useStyles()
-  const onNoClick = () => {
-    onNo()
+  const onYesClick = useCallback(async () => {
+    // Log the search event but time-cap how long we wait to avoid a bad UX
+    // if the request hangs.
+    try {
+      const MS_TO_WAIT_FOR_LOG = 50
```

Comment:
Let's increase this given that the click is a one-time event and the mutation is critical to the UX (makes sure the notification stays hidden). 1,500 ms?